internal/provider/resource_ise_network_access_authentication_rule.go

```diff
@@ -288,6 +289,9 @@ func (r *NetworkAccessAuthenticationRuleResource) Create(ctx context.Context, re
 
 	// Create object
 	body := plan.toBody(ctx, NetworkAccessAuthenticationRule{})
+
+	tflog.Debug(ctx, fmt.Sprintf("Generated body123: %s", body))
```

Comment:
I think this can be removed.

Reply:
yes, my bad forgot to clean it up after fixing some issues i had, removed